api/src/main/java/org/pmiops/workbench/db/model/DbVerifiedInstitutionalAffiliation.java

```diff
@@ -91,15 +95,19 @@ public boolean equals(Object o) {
 
     DbVerifiedInstitutionalAffiliation that = (DbVerifiedInstitutionalAffiliation) o;
 
-    return Objects.equals(user, that.user)
+    // TODO: DbUser doesn't have a well-defined equals() so we use the username instead
```

Comment:
open to suggestions for a better pattern here (and below)

Reply:
Why not add one in `DbUser`? If `equals` is taken, you could use `isEffectivelyEqualTo()` or something.

Reply:
done

Reply:
nope - my attempt at full DbUser.equals() resulted in
```
Caused by: org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: org.pmiops.workbench.db.model.DbUser.authorities, could not initialize proxy - no Session
```
So I chose the easier username-only path